### PR TITLE
Doc how to access our calls on desktop/mobile

### PR DESCRIPTION
**Pull request by patcon**
Re-ticketed from https://github.com/hyphacoop/organizing/issues/127#issuecomment-547047666

How this look @ASoTNetworks?

### CODE REVIEW

**Comment by ASoTNetworks** [comment]
That looks fine to me other than "copy-paste call URL into app home screen" which I have a feeling some might copy "link.hypha.coop/calls" into the app instead of the direct Jitsi link which it will not work.

File: guides.md [hunk 1]
```diff
@@ -70,6 +70,17 @@ members. Skip the steps below at your discretion for low-stakes topics.
...
   - as needed, send reminders of action items
   - migrate action items into task tracker
+  
+### Joining a Jitsi Call
+
+:computer: **Desktop.** Follow [`link.hypha.coop/calls`](https://link.hypha.coop/calls) to our default conference platform.
+
+:iphone: **Mobile**
+  1. Install the app. [Android][jitsi-android] | [iOS][jitsi-ios]
+  2. :link: **Visit** call URL to open app, or :clipboard: **copy-paste** call URL into app home screen.
```
Comment (patcon) [inline guides.md]:
```suggestion
  2. :link: **Visit** call URL to open app, or :clipboard: **copy-paste** redirected call URL into app home screen.
```

Comment (ASoTNetworks) [reply guides.md]: I guess that would work.

**Comment by patcon** [comment]
Cool. Is it alright if I leave this as assigned to you @ASoTNetworks, to merge as docs for your jitsi PR whenever you feel it's done and ready to merge? I wanted to offer something concrete for docs instead of just suggesting ideas, but I'm not trying to take ownership of this :)

**Comment by patcon** [comment]
bump @ASoTNetworks

File: guides.md [hunk 2]
```diff
@@ -70,6 +77,17 @@ members. Skip the steps below at your discretion for low-stakes topics.
...
   - as needed, send reminders of action items
   - migrate action items into task tracker
+  
+### Joining a Jitsi Call
+
+💻 **Desktop.** Follow [`link.hypha.coop/calls`](https://link.hypha.coop/calls) to our default conference platform.
+
+📱 **Mobile**
+  1. Install the app. [Android][jitsi-android] | [iOS][jitsi-ios]
+  2. 🔗 **Visit** call URL to open app, or 📋 **copy-paste** redirected call URL into app home screen.
```
Comment (dcwalk) [inline guides.md]:
```suggestion
  2. 🔗 **Visit** call URL to open app, or 📋 **copy-paste** redirected call URL (e.g. `https://jitsi.hypha.coop/hyphacoop`) into app home screen.
```

Comment (dcwalk) [reply guides.md]: Does this address your comment @ASoTNetworks ?

Comment (ASoTNetworks) [reply guides.md]: Yes that looks good.

**Comment by patcon** [comment]
Pls do any actions and merge if looks good @ASoTNetworks. This ticket is yours to close out

**Pull request merged**